### ISSUE TITLE
fix docker image build - proxy dep

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2861]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.44.8](https://github.com/open-wa/wa-automate-nodejs/compare/4.44.7...4.44.8) (2022-09-01)

- fix #2861 [`#2861`](https://github.com/open-wa/wa-automate-nodejs/issues/2861)
- Release 4.44.8 [`ff08fab`](https://github.com/open-wa/wa-automate-nodejs/commit/ff08fab3f47cc1dfe1839d0b97aa8db74d9913f2)
- updated types-only package [`b3722e9`](https://github.com/open-wa/wa-automate-nodejs/commit/b3722e92a0ad0980828ace4b442932a7240f3d98)

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.44.9](https://github.com/open-wa/wa-automate-nodejs/compare/4.44.8...4.44.9) (2022-09-01)

- fix #2861 [`#2861`](https://github.com/open-wa/wa-automate-nodejs/issues/2861)
- fix #2861 [`#2861`](https://github.com/open-wa/wa-automate-nodejs/issues/2861)
- Release 4.44.9 [`7313988`](https://github.com/open-wa/wa-automate-nodejs/commit/73139886ff0b3e633b7962896c8c63eeacec3579)
- updated types-only package [`f2cb8a8`](https://github.com/open-wa/wa-automate-nodejs/commit/f2cb8a88cb2cbccdfdeef40ed2251ff27e82905a)
